pandas/tests/plotting/test_series.py

```diff
@@ -977,6 +974,25 @@ def test_series_none_color(self):
         expected = _unpack_cycler(mpl.pyplot.rcParams)[:1]
         _check_colors(ax.get_lines(), linecolors=expected)
 
+    @pytest.mark.parametrize(
```

Comment:
You don't need parametrize here since you are only testing one case; instead you can just construct the DataFrame in the test body with these values directly